deployment/postgres-init-db/sql/tables/scheduler/scheduling_run/scheduling_goal_analysis.sql

```diff
@@ -1,11 +1,13 @@
 create table scheduler.scheduling_goal_analysis (
   analysis_id integer not null,
   goal_id integer not null,
+  goal_invocation_id integer not null,
```

Comment:
Minor: This column is missing a comment.

Reply:
Fixed